src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -65,11 +79,22 @@ public async Task WarmupInvoke(HttpContext httpContext)
 
             ReadRuntimeAssemblyFiles();
 
-            await WarmUp(httpContext.Request);
+            await WorkerWarmup();
+
+            await HostWarmUp(httpContext.Request);
```

Comment:
Shouldn't we be doing HostWarmup first then WorkerWarmup next?

nit: HostWarmUp --> HostWarmup